core/src/com/projectkorra/projectkorra/ability/FireAbility.java

```diff
@@ -168,6 +169,30 @@ public static BlockData createFireState(Block position, boolean blue) {
 		return fire;
 	}
 
+	public static void dryWetBlocks(final Block block, final CoreAbility ability, boolean playSound) {
+		if (GeneralMethods.isRegionProtectedFromBuild(ability, block.getLocation())) {
+			return;
+		}
+		if (block.getType() == Material.WET_SPONGE) {
+			block.setType(Material.SPONGE);
+
+			if (playSound) {
+				block.getWorld().playSound(block.getLocation(), Sound.BLOCK_FIRE_EXTINGUISH, 0.5F, 1);
+			}
+		} else if (isSnow(block)) {
+			block.getWorld().spawnParticle(Particle.BLOCK_DUST, block.getLocation().add(0.5, 0.5, 0.5), 2, 0.5, 0.5, 0.5, 0.1, Material.SNOW_BLOCK.createBlockData());
+			block.setType(Material.AIR);
+
+			if (playSound) {
+				block.getWorld().playSound(block.getLocation(), Sound.BLOCK_SNOW_BREAK, 1, 1);
+			}
+		}
```

Comment:
Similar to the water transform blocks, maybe we should do a fire trasform blocks or something here?

Reply:
Transformable blocks were for water sourcing, I don't see why we should make fire abilities transform blocks (here we're just drying sponges and snow). Maybe to refactor some code since we do power furnaces and campfires but that's about it.